code/web/src/modules/user/api/state.js

```diff
@@ -43,6 +43,14 @@ export default (state = userInitialState, action) => {
         details: null
       }
 
+    case UPDATE_USER:
```

Comment:
this should possibly be deleted. I don't think we're using it anywhere at this point.